### DISCUSSION TITLE
A more memorizable name / Change Name

### DISCUSSION THREAD

**Discussion by maverick74** [lxqt/lxqt#2175] [Ideas]
OK...
I totally know i'm going to get hanged by this... You can go pick up your torches and forks and get ready for a chase, now!

How about...

**Changing LXQt project name?**

Here's the thing:

L stands for Lite
X for X.org
Qt for the toolkit

Now, we are already talking about LWQt were W means Wayland.
And it already changed from LXDE > LXQt (yes, i know its a fork and all the other bla, bla, bla details...)
This is all fine for techie guys, but not for the regular user.

Something that would represent well the project would be something like **LiteDE** (it's an example, other suggestions are welcome).
I suggest this name because that's what we aim for: a Lite Desktop Enviroment.
it's a simple and direct name that easily translates what LXQt is: the lightest DE (the litest & decent there is, at least)

so... here's my idea :)

(_OK... now i go hide myself somewhere! ahah_)

**Comment by tsujan** [comment]
> L stands for Lite...

LXQt is just a name and I don't think any LXQt member wants to change it.

People have different theories about what "K" stands for in "KDE" but it's just "K". Also "GNU Network Object Model Environment" is a relic of old times and "GNOME" is just "GNOME"; etc.

> And it already changed from LXDE > LXQt

It didn't. LXQt is a separate project.

> Now, we are already talking about LWQt

As far as I know, no LXQt member said anything about "LWQt".

**Reply by luis-pereira** [reply]
The man hours needed to change from LXQt to something else would be better used developing LXQt.
The `X` in LXQt, IMO, comes from Unix.
That's my two cents.

**Reply by stefonarch** [reply]
I don't think it's  good idea to change the name for LXQt on X11, but frankly LWQt . continuing just the naming scheme . doesn't convince me much. 
It sounds weird, so if ever we have a wayland version running a   better name could  be found IMHO.

**Reply by tsujan** [reply]
> if ever we have a wayland version running a better name could be found IMHO.

No new name will be needed by a Wayland LXQt session, as it isn't needed by other DEs.

**Reply by maverick74** [reply]
( _getting out of my cave and walking cautiously_ )
Don't get me wrong, it's not that i don't like the name - i'm very used to it and @luis-pereira (_olá Luís!_) does have it's reason in saying that  the time "_would be better used developing LXQt._".

Just wanted to send the subject to the table because, while i do like LXQt, it actually does not "mean" a thing to anyone new to Linux. Hence suggesting a "clear" name.

@tsujan 
> People have different theories about what "K" stands for in "

If i recall this correctly, Matthias intended it to mean Kool Desktop Environment... but yes that was back in the day and today it's just the Community name

> > And it already changed from LXDE > LXQt
> 
> It didn't. LXQt is a separate project.

that's the "_bla bla bla_" part XP (but almost everybody outside of the dev teams see it as the _spiritual_ sucessor of LXDE - most don't even recall it's also based on RazorQt)... 
But, naturally as a developer, in this case, you are undoubtedly correct as you are inside the team and know the code. So i can't argue this. But... i'm sure you understand what i try to transmit... everybody sees a link (even if there isn't one!)

**Reply by tsujan** [reply]
> that's the "bla bla bla" part XP

It can be said that LXQt was forked from RazorQt.

I wasn't an LXDE user (I used KDE before discovering LXQt) but I think that the only programs that could be considered forked from LXDE were pcmanfm and lximage. Their Qt counterparts used LXDE's libfm for a while, and that caused lots of troubles. Later, libfm-qt was created for wrapping GLib/GIO functions in C++11 and we were free to develop pcmanfm-qt.

Anyway, names are just names. IMO, even if we had a good reason and also the required manpower, changing the name of a project like LXQt and saying goodbye to its history wouldn't be wise.

**Reply by maverick74** [reply]
> Anyway, names are just names. IMO, even if we had a good reason and also the required manpower, changing the name of a project like LXQt and saying goodbye to its history wouldn't be wise.

I see what you mean, but i don't think you would "loose it's history"... maybe just brand name (like LibreOffice had to build its name from the ground up)

(Off topic: Wouldn't GLib/GIO's KDE counterparts make more sense to use in a Qt environment or are they not as capable?)

**Reply by tsujan** [reply]
> Wouldn't GLib/GIO's KDE counterparts make more sense to use in a Qt environment or are they not as capable?

It's a long story but, in short, Qt's file management is minimal and KDE's isn't as capable as GLib's. Like any other program, GLib has its flaws — and bugs — but it has a high quality.

**Reply by luis-pereira** [reply]
> It's a long story but, in short, Qt's file management is minimal and KDE's isn't as capable as GLib's. Like any other program, GLib has its flaws — and bugs — but it has a high quality.

GLIb and Gnome are different beasts. GLIb is high quality.
